pkg/k8shandler/visualization.go

```diff
@@ -41,79 +45,62 @@ var (
 func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateVisualization(proxyConfig *configv1.Proxy) (err error) {
 	if clusterRequest.cluster.Spec.Visualization == nil || clusterRequest.cluster.Spec.Visualization.Type == "" {
 		if err = clusterRequest.removeKibana(); err != nil {
+			logrus.Error(err.Error())
```

Comment:
nit -- we don't need to print the error here because it gets printed out eventually when we return it